codespell_lib/data/dictionary.txt

```diff
@@ -10226,6 +10229,7 @@ paintin->painting
 paitience->patience
 paiting->painting
 pakage->package
+pakageimpl->packageimpl
```

Comment:
This is highly specialized. Thought I don't see a major problem having it in the dictionary.